src/praisonai-bot/praisonai_bot/scheduler/executor.py

```
@@ -461,6 +481,131 @@ async def _execute_one(self, job: "ScheduleJob") -> JobResult:
+        delivery = getattr(job, "delivery", None)
+        if text and delivery and self._deliver:
+            try:
+                coro = self._deliver(delivery, text)
+                if inspect.isawaitable(coro):
+                    await coro
+                delivered = True
+                logger.info(
+                    "Delivered command job '%s' output to %s:%s",
+                    job.id, delivery.channel, delivery.channel_id,
+                )
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)